wiki/Contests/Official_support/en.md

```diff
@@ -22,6 +22,7 @@ Officially supported contests have access to certain benefits:
 
 - **Profile badge prizes for first-place winners.** See profile badge design criteria on the [official tournament support article](/wiki/Tournaments/Official_support#profile-badges).
 - **Progress towards *Elite Mapper* user title.** See [contest points](/wiki/Contests/Contest_points) for more details.
+- **Potential consideration for [news coverage](#news-posts)** (outside the required coverage if the contest is awarding [contest points](/wiki/Contests/Contest_points)).
```

Comment:
aren't these the same? i.e. if the contest is covered in news -> can go for contest points

Reply:
yeah but its not implied anywhere that you can apply for news coverage even though you dont intend to go for contest points hence the disclaimer

Reply:
ah, then this should be clarified in the section about contest points

```suggestion
- **Potential consideration for [news coverage](#news-posts)**.
```